docs/app/client/carousel/descriptions/SelectIndex.md

```diff
@@ -2,4 +2,4 @@
 title: Selected Index
 ---
 
-The `selectedIndex` prop enables the carousel to display a particular index.
+The `selectedIndex` prop enables the carousel to display a particular index. Add `isSwipeDragDisabled` to disable swipe/drag over the carousel content (optional).
```

Comment:
Potential breaking change. Mentioned in AutoSlide example

Reply:
Removed.